drizzle/tests/test_drizzle.py

```diff
@@ -251,6 +251,36 @@ def test_square_with_point(tmpdir):
         assert max_diff < 1.0e-5
 
 
+def test_zero_input_weight(tmpdir):
+    """
+    Test do_driz square kernel with grid
+    """
+    insci = np.ones((200, 400), dtype=np.float32)
+    inwht = np.ones((200, 400), dtype=np.float32)
+    inwht[:, 150:153] = 0
+    pixmap = np.rollaxis(np.mgrid[1:201, 1:401][::-1], 0, 3)
```

Comment:
rollaxis is deprecated, with moveaxis being the preferred replacement.

Reply:
Thanks for pointing this out. Will update the code soon.